Automattic/jetpack
# Update credentials setup message

We're updating the message shown when credentials are not set up in Calypso blue with the same with we have in Calypso green, for feature parity and to avoid customers contacting support because they think credentials are needed for backups to work.





Fixes #

#### Changes proposed in this Pull Request:

* We changed the title and message in the settings card to clarify that server credentials are needed for restores. 

#### Other information:


#### Jetpack product discussion
p7pQDF-7co-p2#comment-21470

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Purchase a Jetpack Backup plan
* Go to the Jetpack dashboard "At a glance" section
* Check that the text Backup card under Security section is: "Enter your SSH, SFTP or FTP credentials to enable one-click site restores and faster backups"
* Go to the Security settings page
* Check that the text is: "Add SSH, SFTP, or FTP credentials to enable one-click site restores"
* Purchase a Jetpack Scan plan
* Go to the Jetpack dashboard "At a glance" section
* Check that the text is: "Add SSH, SFTP, or FTP credentials to enable one-click fixes"
* Go to the Security settings page
* Check that the text under the Security section is: "Add SSH, SFTP, or FTP credentials to enable one-click site restores and fixes"
